src/App.css

```diff
@@ -8,24 +8,17 @@
 }
 
 body {
-  background: url("../assets/library.jpg");
-  background-color: rgba(1, 10, 8, 0.4);
-  background-blend-mode: darken;
-  background-repeat: no-repeat;
-  background-size: 100% 100vh;
-  color: white;
+  background-color: #eae3e3;
   font-family: "Quicksand", sans-serif;
 }
```

Comment:
- [ ] Good job with styling 🥇🥇 Please consider using the same background color for `body` as given in the Zeplin. You are using `#eae3e3` but in Zeplin, it is `#fafafa` ⚒ Please fix this ✨

Reply:
Ok...thanks for the suggestion.